### ISSUE TITLE
Missing a keybind comment in default keybinding config?

### ISSUE THREAD

**Issue by amackenzie** [PeterTh/gedosato#345]
On the 2014-11-02 blog post, you mention that there's a keybind to reload settings for testing out PSHashes and such, but the default GeDoSaToKeys.ini doesn't have that listed, nor does the README.txt file.

Is there another place where it may be listed that I may have missed? I may have just overlooked something.

**Comment by PeterTh** [comment]
Yes, both settings and keybindings are missing from their respective lists. I need to update them, or better yet, generate them from code :P

In the meantime, you can find all actions here:
https://github.com/PeterTh/gedosato/blob/master/defs/actions.def

and all settings here:
https://github.com/PeterTh/gedosato/blob/master/defs/settings.def

(do note that some actions/settings might not be documented on purpose, because they are broken and/or unstable)
